server/config.js

```diff
@@ -6,7 +6,8 @@ const cache = new Map()
 const defaultConfig = {
   webpack: null,
   poweredByHeader: true,
-  distDir: '.next'
+  distDir: '.next',
+  assetPrefix: ''
```

Comment:
no, I don't think we _need_ it. #907 had it, so I figured we'd follow suit here. I suppose since `distDir` is a string, `assetPrefix` should be too tho.